docs/permissions/limitation_reference.md

```diff
@@ -42,6 +42,24 @@ Out of the box FunctionList uses it in the following way:
             - {name: ibexa.permissions.limitation_type, alias: FunctionList}
 ```
 
+## Change Owner Limitation
+
+A Limitation to specify whether the user can change the owner of a Content item.
+
+|                 |                                                                                                |
+|-----------------|------------------------------------------------------------------------------------------------|
+| Identifier      | `ChangeOwner`                                                                                  |
+| Value Class     | `Ibexa\Contracts\Core\Repository\Values\User\Limitation\ChangeOwnerLimitation`                 |
+| Type Class      | `Ibexa\Core\Limitation\ChangeOwnerLimitationType`                                              |
+| Criterion used  | `Ibexa\Contracts\Core\Repository\Values\Content\Query\Criterion\UserMetadata(UserMetadata::OWNER )` |
+| Role Limitation | no                                                                                             |
```

Comment:
What does this "no" mean, again?

Reply:
It means the Limitation can be set only on a Policy, but not on a Role assignment, like here: 
![image](https://user-images.githubusercontent.com/13621049/200327279-99f93be5-963c-415d-81d4-04a0d48934d4.png)

Reply:
Ah right, thanks :D